tests/app/questionnaire/test_router.py

```diff
@@ -646,6 +647,145 @@ def test_return_to_calculated_summary_return_to_block_id_not_on_path(self):
             == next_location_url
         )
 
+    @pytest.mark.usefixtures("app")
```

Comment:
Do you need all these test?. These all exist for the old format, so you can just parametrise the schema?. I think you made the IDs between the two the same, so apart from the schema that is loaded, it should be identical?.